tests/wicked/basic/ref/t08_setup_second_card.pm

```diff
@@ -22,11 +22,15 @@ use lockapi;
 sub run {
     my ($self, $ctx) = @_;
     record_info('Info', 'Set up a second card');
-    assert_script_run(sprintf("ip a a %s/15 dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
+    assert_script_run(sprintf("ip a a %s/24 dev %s", $self->get_ip(type => 'dhcp_2nic'),   $ctx->iface()));
+    assert_script_run(sprintf("ip a a %s/24 dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
     systemctl 'stop dhcpd.service';
     $self->get_from_data('wicked/dhcp/dhcpd_2nics.conf', '/etc/dhcpd.conf');
     systemctl 'start dhcpd.service';
     $self->wait_for_dhcpd();
+    # Enable forwarding to reach the gateway from SUT
+    assert_script_run('sed -i s/0/1/g /etc/sysctl.conf');
```

Comment:
replace **all** zeros in file with ones ? looks fishy , can we be more specific ?

Reply:
yes, basically it will change from
```
net.ipv4.ip_forward = 0
net.ipv6.conf.all.forwarding = 0
```
to
```
net.ipv4.ip_forward = 1
net.ipv6.conf.all.forwarding = 1
```

Reply:
cool but what if there are more lines ?

Reply:
why not just  do `sysctl -w net.ipv4.ip_forward = 1` and `sysctl -w net.ipv6.conf.all.forwarding = 1 `